docs/response-parameters.rst

```diff
@@ -51,10 +51,12 @@ Descriptions of the custom response parameters that are marked with a leading ``
 * ``@changesetId`` - id of the OSM changeset where the contribution was performed
 * ``@osmType`` - type of the OSM element (NODE, WAY or RELATION)
 * ``@timestamp`` - indicates when this contribution occurred
-* ``@creation``	- contribution type; indicates if this feature is newly created (true); cannot occur in combination with other contribution types
+* ``@creation``	- contribution type; indicates if a new feature gets created OR moved into your requested area of interest OR is now fitting to your defined filter parameter (true); cannot occur in combination with other contribution types
```

Comment:
:arrow_down: please replace the tab with a space, and take a look at the tweaked wording:

```suggestion
* ``@creation`` - contribution type; indicates if the OSM element newly fits the query's requirements (either because it is newly created, moved into the query area of interest, or is now fitting to the defined filter parameter); cannot occur in combination with other contribution types
```

Reply:
Nice catch. You can't see that tab anywhere else but in here on GitHub :D neither in the editor that I use to edit the .rst files, nor in the html. 

About your suggested wording: I think we should also change ``feature`` with ``element`` as we've used this word also in other parts here. And I think it should also say ``.. query's area of interest..``. Apart from that, it looks fine.